changelogs/fragments/8073-ldap-attrs-diff.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ldap_attrs - Module now supports diff mode, showing which attributes are changed within an operation
```

Comment:
Please check the documentation and other fragment files as examples. We are missing the link to this PR (and the issue url if there was one).

Reply:
Yup, has been adapted. Only skimmed thorough the docs, so I missed that there should be a reference to the PR.
There was no issue to this, felt like that would be pointless if it's just for the sake creating a PR right after.